include/oneapi/tbb/concurrent_vector.h

```diff
@@ -644,6 +644,23 @@ class concurrent_vector
         }
     }
 
+    segment_type nullify_segment( segment_table_type table, size_type segment_index ) {
+        segment_type target_segment{nullptr};
```

Comment:
Why does not make it as you suggested in the resolved conversation:

```suggestion
        segment_type target_segment = table[segment_index].load(relaxed);
        table[segment_index].store(nullptr, relaxed);
        if (segment_index == 0) {
            for (size_type i = 1; i < my_first_block; ++i) {
                table[i].store(nullptr, relaxed);
        }
        return target_segment;
```